frontend/packages/design-system/src/assets/icons/ActivityCamp.tsx

```diff
@@ -1,13 +1,11 @@
 import React from 'react';
 import { Path } from 'react-native-svg';
 
-import { IconVariants } from '../../types/icons';
-
-export default (variant: IconVariants) => {
-  return (
-    <Path
-      fill={variant}
-      d="M12,18.2l8-4.6l8,4.6c0.2,0.1,0.4,0.2,0.6,0.2c0.4,0,0.8-0.2,1.1-0.6c0.3-0.6,0.1-1.3-0.5-1.7l-8.6-4.9
+export const ActivityCamp = (
+  <Path
+    fill="currentColor"
+    stroke="currentColor"
```

Comment:
I don't think we want to set `fill` and `stroke`. When the designer makes an icon they will set which property uses the colour. We should just replace that colour with `currentColor`. Here it would be `fill`.